### PR TITLE
Proofread

### PR DESCRIPTION
**Pull request by verretor**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -312,9 +312,9 @@ eclair-cli forceclose --channelId=<channel>
 }
 ```
 
-Initiates an unilateral close for given channels that belong to this eclair node, once the commitment has been broadcasted the API returns its
+Initiates a unilateral close for given channels that belong to this eclair node. Once the commitment has been broadcasted, the API returns its
```
Comment (t-bast) [inline source/index.html.md:315]:
```suggestion
Initiates a unilateral close for given channels that belong to this eclair node. Once the commitment has been broadcast, the API returns its
```

File: source/index.html.md [hunk 2]
```diff
@@ -359,8 +359,8 @@ eclair-cli updaterelayfee \
 }
 ```
 
-Updates the fee policy for the specified _channelId_, a new update for this channel will be broadcasted to the network.
-The endpoint supports receiving multiple channel id(s) or short channel id(s); to update multiple channel you can use the parameters `channelIds` or `shortChannelIds` below.
+Updates the fee policy for the specified _channelId_. A new update for this channel will be broadcasted to the network.
```
Comment (t-bast) [inline source/index.html.md:362]:
```suggestion
Updates the fee policy for the specified _channelId_. A new update for this channel will be broadcast to the network.
```

**Review by t-bast — COMMENTED** [review]
Thanks, there were indeed a lot of small things to fix!

**Comment by verretor** [comment]
Done.

**Review by t-bast — APPROVED** [review]
Thanks!

**Pull request merged**